.github/workflows/continuous-delivery.yaml

```diff
@@ -39,6 +39,16 @@ jobs:
           commit-assets: |
             ./dist
           node-module: true
+          release-rules: |
+            [
+              { "release": "patch", "type": "build" },
+              { "release": "patch", "type": "chore", "scope": "deps" },
+              { "release": "patch", "type": "chore", "scope": "deps-dev", "message": "bump typescript from*" },
```

Comment:
At this moment simple chores like `chore: cleanup depedencies` dont get a release.

Reply:
Fixed!

Reply:
Actually, now it should be fixed! 😬